packages/node/src/node.ts

```diff
@@ -1,12 +1,8 @@
-import EventEmitter from "eventemitter3";
-
 import {
   AppInstanceInfo,
-  GetAppInstancesResult,
-  MethodName,
-  MethodRequest,
-  MethodResponse
-} from "./node-types";
+  Node as NodeTypes
```

Comment:
@cf19drofxots Couldn't really come up with an elegent name for this. Can't call it "Node" because that would conflict with the Node class. Any suggestions?

Reply:
Mkay